packages/hydrooj/locales/ko_KR.yaml

```diff
@@ -465,7 +465,7 @@ Only A-Z, a-z, 0-9 and _ are accepted: A-Z, a-z, 0-9 와 _만 허용됨
 Oops, there are no results.: 결과 없음
 Oops!: Oops!
 Open in Problem Set: 문제에서 열기
-Open Scratchpad: 연습장
+Open Scratchpad: 제출창
```

Comment:
maybe it's better to translate it to "Online IDE" ? or "Open Editor" ?

Reply:
I think "Open Editor" will be better.